File: javascript/high-scores/high-scores.js
export class HighScores {
  #scores
  constructor(scores) {
    this.#scores = scores;
  }

  get scores() {
    return this.#scores;
  }

  get latest() {
    return this.#scores.slice(-1)[0];
  }

  get personalBest() {
    return Math.max(...this.#scores);
  }

  get personalTopThree() {
    return [...this.#scores].sort((a, b) => b - a).slice(0, 3);
  }
}

File: python/high-scores/high_scores.py
class HighScores:
    def __init__(self, scores):
        self.scores = scores
        self.scores_sorted = sorted(scores, reverse=True)
        print(self.scores_sorted)

    def personal_top_three(self):
        return self.scores_sorted[0:3]

    def personal_best(self):
        return self.scores_sorted[0]

    def latest(self):
        return self.scores[-1]
